[2023-06-24T09:04:22.802+0700] {processor.py:157} INFO - Started process (PID=1567) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:04:22.802+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T09:04:22.803+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:04:22.803+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:04:22.820+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:04:22.820+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 18, in <module>
    start_date=datetime(2023,27,6),
ValueError: month must be in 1..12
[2023-06-24T09:04:22.820+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:04:23.247+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.445 seconds
[2023-06-24T09:07:26.393+0700] {processor.py:157} INFO - Started process (PID=1633) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:07:26.393+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T09:07:26.394+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:07:26.393+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:07:26.419+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:09:26.035+0700] {processor.py:157} INFO - Started process (PID=1665) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:09:26.035+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T09:09:26.036+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.036+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:09:26.061+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:09:26.548+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.548+0700] {manager.py:504} INFO - Created Permission View: can delete on DAG:test_alpha_team
[2023-06-24T09:09:26.555+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.555+0700] {manager.py:504} INFO - Created Permission View: can read on DAG:test_alpha_team
[2023-06-24T09:09:26.560+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.560+0700] {manager.py:504} INFO - Created Permission View: can edit on DAG:test_alpha_team
[2023-06-24T09:09:26.568+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.568+0700] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-24T09:09:26.575+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.575+0700] {dag.py:2747} INFO - Creating ORM DAG for test_alpha_team
[2023-06-24T09:09:26.581+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:09:26.581+0700] {dag.py:3486} INFO - Setting next_dagrun for test_alpha_team to 2023-06-27T00:00:00+00:00, run_after=2023-06-28T00:00:00+00:00
[2023-06-24T09:09:26.592+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.558 seconds
[2023-06-24T09:10:01.864+0700] {processor.py:157} INFO - Started process (PID=1689) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:10:01.865+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T09:10:01.865+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:10:01.865+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:10:01.893+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:10:02.342+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:10:02.342+0700] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-24T09:10:02.356+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:10:02.356+0700] {dag.py:3486} INFO - Setting next_dagrun for test_alpha_team to 2023-06-27T00:00:00+00:00, run_after=2023-06-28T00:00:00+00:00
[2023-06-24T09:10:02.370+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.506 seconds
[2023-06-24T09:30:44.767+0700] {processor.py:157} INFO - Started process (PID=1881) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:30:44.768+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T09:30:44.768+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:30:44.768+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:30:44.789+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:30:44.788+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 28, in <module>
    task2 = PythonOperator(
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/operators/python.py", line 166, in __init__
    super().__init__(**kwargs)
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 799, in __init__
    validate_key(task_id)
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/utils/helpers.py", line 55, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'python airflows' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2023-06-24T09:30:44.789+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:30:45.209+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.443 seconds
[2023-06-24T09:42:39.857+0700] {processor.py:157} INFO - Started process (PID=2001) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:42:39.858+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T09:42:39.858+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:42:39.858+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:42:39.883+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T09:42:40.378+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:42:40.377+0700] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-24T09:42:40.389+0700] {logging_mixin.py:149} INFO - [2023-06-24T09:42:40.389+0700] {dag.py:3486} INFO - Setting next_dagrun for test_alpha_team to 2023-06-27T00:00:00+00:00, run_after=2023-06-28T00:00:00+00:00
[2023-06-24T09:42:40.403+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.546 seconds
[2023-06-24T10:35:52.679+0700] {processor.py:157} INFO - Started process (PID=2429) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:35:52.679+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T10:35:52.680+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:35:52.680+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:35:52.699+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:35:52.699+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 8, in <module>
    import create_dim_fact_tb
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/create_dim_fact_tb.py", line 1, in <module>
    import psycopg2
ModuleNotFoundError: No module named 'psycopg2'
[2023-06-24T10:35:52.700+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:35:53.145+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.466 seconds
[2023-06-24T10:37:49.857+0700] {processor.py:157} INFO - Started process (PID=2466) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:37:49.858+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T10:37:49.858+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:37:49.858+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:37:49.876+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:37:49.875+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 7, in <module>
    import psycopg2
ModuleNotFoundError: No module named 'psycopg2'
[2023-06-24T10:37:49.876+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:37:50.314+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.457 seconds
[2023-06-24T10:39:18.880+0700] {processor.py:157} INFO - Started process (PID=2507) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:39:18.880+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T10:39:18.881+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:39:18.880+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:39:18.900+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:39:18.900+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 4, in <module>
    import psycopg2
ModuleNotFoundError: No module named 'psycopg2'
[2023-06-24T10:39:18.901+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:39:19.336+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 0.457 seconds
[2023-06-24T10:41:12.469+0700] {processor.py:157} INFO - Started process (PID=2572) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:41:12.470+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T10:41:12.470+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:41:12.470+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:41:12.681+0700] {logging_mixin.py:149} INFO - Bắt đầu tiến trình tạo datamart
[2023-06-24T10:41:12.999+0700] {logging_mixin.py:149} INFO - Hoàn tất tiến trình tạo datamart
[2023-06-24T10:41:13.013+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:41:13.585+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:41:13.585+0700] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-24T10:41:13.596+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:41:13.596+0700] {dag.py:3486} INFO - Setting next_dagrun for test_alpha_team to 2023-06-27T00:00:00+00:00, run_after=2023-06-28T00:00:00+00:00
[2023-06-24T10:41:13.610+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 1.141 seconds
[2023-06-24T10:52:55.869+0700] {processor.py:157} INFO - Started process (PID=2657) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:52:55.870+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T10:52:55.870+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:52:55.870+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:52:56.057+0700] {logging_mixin.py:149} INFO - Bắt đầu tiến trình tạo datamart
[2023-06-24T10:52:56.619+0700] {logging_mixin.py:149} INFO - Hoàn tất tiến trình tạo datamart
[2023-06-24T10:52:56.632+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:52:57.100+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:52:57.100+0700] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-24T10:52:57.113+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:52:57.113+0700] {dag.py:3486} INFO - Setting next_dagrun for test_alpha_team to 2023-06-27T00:00:00+00:00, run_after=2023-06-28T00:00:00+00:00
[2023-06-24T10:52:57.123+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 1.255 seconds
[2023-06-24T10:59:41.019+0700] {processor.py:157} INFO - Started process (PID=2742) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:59:41.019+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T10:59:41.020+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:59:41.020+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:59:41.233+0700] {logging_mixin.py:149} INFO - Bắt đầu tiến trình tạo datamart
[2023-06-24T10:59:41.591+0700] {logging_mixin.py:149} INFO - Hoàn tất tiến trình tạo datamart
[2023-06-24T10:59:41.735+0700] {logging_mixin.py:149} INFO - [2023-06-24T10:59:41.733+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 39, in <module>
    task2 = PythonOperator(
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/operators/python.py", line 168, in __init__
    raise AirflowException("`python_callable` param must be callable")
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2023-06-24T10:59:41.735+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T10:59:42.185+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 1.167 seconds
[2023-06-24T11:01:47.143+0700] {processor.py:157} INFO - Started process (PID=2778) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T11:01:47.143+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T11:01:47.144+0700] {logging_mixin.py:149} INFO - [2023-06-24T11:01:47.144+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T11:01:47.544+0700] {logging_mixin.py:149} INFO - Bắt đầu tiến trình tạo datamart
[2023-06-24T11:01:48.198+0700] {logging_mixin.py:149} INFO - Hoàn tất tiến trình tạo datamart
[2023-06-24T11:01:48.594+0700] {logging_mixin.py:149} INFO - [2023-06-24T11:01:48.592+0700] {dagbag.py:350} ERROR - Failed to import: /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
Traceback (most recent call last):
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py", line 38, in <module>
    task2 = PythonOperator(
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/myenv/lib/python3.9/site-packages/airflow/operators/python.py", line 168, in __init__
    raise AirflowException("`python_callable` param must be callable")
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2023-06-24T11:01:48.594+0700] {processor.py:838} WARNING - No viable dags retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T11:01:49.045+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 1.903 seconds
[2023-06-24T11:03:29.374+0700] {processor.py:157} INFO - Started process (PID=2813) to work on /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T11:03:29.374+0700] {processor.py:826} INFO - Processing file /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py for tasks to queue
[2023-06-24T11:03:29.375+0700] {logging_mixin.py:149} INFO - [2023-06-24T11:03:29.374+0700] {dagbag.py:541} INFO - Filling up the DagBag from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T11:03:29.599+0700] {logging_mixin.py:149} INFO - Bắt đầu tiến trình tạo datamart
[2023-06-24T11:03:29.944+0700] {logging_mixin.py:149} INFO - Hoàn tất tiến trình tạo datamart
[2023-06-24T11:03:30.088+0700] {processor.py:836} INFO - DAG(s) dict_keys(['test_alpha_team']) retrieved from /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py
[2023-06-24T11:03:30.604+0700] {logging_mixin.py:149} INFO - [2023-06-24T11:03:30.604+0700] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-24T11:03:30.615+0700] {logging_mixin.py:149} INFO - [2023-06-24T11:03:30.615+0700] {dag.py:3486} INFO - Setting next_dagrun for test_alpha_team to 2023-06-27T00:00:00+00:00, run_after=2023-06-28T00:00:00+00:00
[2023-06-24T11:03:30.629+0700] {processor.py:179} INFO - Processing /Users/IntelTri/WorkPlace/Aptech/Project-sem2/project-sem02/air_flows/data_pipline.py took 1.256 seconds
